server/auth.filters.js

```diff
@@ -69,11 +69,11 @@ const assertAdmin = (req, res, next) => {
   if (!req.user) {
     const err = new Error('You must be logged in')
     err.status = 401
-    throw err
+    next(err)
```

Comment:
I think next is better to use in express from what I'm looking at online, but we still need an error handler to deal with it.